checks.d/etcd.py

```diff
@@ -88,6 +76,20 @@ def check(self, instance):
                 else:
                     self.log.warn("Missing key {} in stats.".format(key))
 
+        storeResponse = self.get_store_metrics(url, timeout)
```

Comment:
Already does, stuff like:
```
C:  2, 4: Invalid variable name "snakeCaseVar" (invalid-name)
```
However, we don't throw pylint errors at this level (convention) otherwise I guess we would probably have 2 zillions errors to fix.
Writing up about stricter rules & guidelines for the agent code is part of this quest...